pkg/task/restore.go

```diff
@@ -211,6 +221,7 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 		err = restore.SplitRanges(ctx, client, rangeBatch, rewriteRules, updateCh)
 		if err != nil {
 			log.Error("split regions failed", zap.Error(err))
+			// If any error happened, return now, don't execute checksum.
 			return err
```

Comment:
no need to `restorePostWork`?

Reply:
It's now `defer`ed.

Reply:
@overvenus  But this means the `restorePostWork` is done _after_ checksum